core/store/src/db.rs

```diff
@@ -732,11 +732,11 @@ fn rocksdb_read_options() -> ReadOptions {
     read_options
 }
 
-fn rocksdb_block_based_options(cache_size: usize) -> BlockBasedOptions {
+fn rocksdb_block_based_options(len: usize) -> BlockBasedOptions {
```

Comment:
Why we change name of the variable? IMO `cache_size` or `block_cache_size` (because we have many different caches) is more understandable.

Reply:
@Longarithm I changed it back.